OPERATING.md

```diff
@@ -0,0 +1,151 @@
+# Operating
+
+This document contains requirements and tips for operating the adaptor in a production environment.
+
+# AMQP Message Broker Requirements
+
+* The broker must be configured with a limited number of retries and dead-letter queues
+* It is the responsibility of the GP supplier to configure adequate monitoring against the dead-letter queues that allows ALL undeliverable messages to be investigated fully.
```

Comment:
Being very picky, but remove the trailing full stop here (consistency with the other list items)

Reply:
I was also thinking about doing this... or adding the full stops for consistency :D